types/youtube-dl/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for youtube-dl 1.12
+// Project: https://www.npmjs.com/package/youtube-dl
+// Definitions by: Bogdan Surai <https://github.com/bsurai>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import 'node';
```

Comment:
Prefer `/// <reference types="node" />`.

Reply:
@andy-ms Thank you. All comments are fixed.